common-arcana.lic

```diff
@@ -560,21 +560,20 @@ module DRCA
       data['cast'] = "cast #{cast_on}"
       return data
     end
-    echo 'Could not find any planets to cast on'
+    echo "No planet available to cast #{data['abbrev']}"
     nil
   end
 
   def set_moon_data(data)
     return data unless data['moon']
 
-    check_moonwatch
-    # Look for a moon that will be up for the next few minutes
-    moon = UserVars.moons.find { |moon_name, moon_data| UserVars.moons['visible'].include?(moon_name) && moon_data['timer'] >= 4 }.first
```

Comment:
The new `visible_moons` method handles parsing  `UserVars.moons` and calling `check_moonwatch`